worlds/sc2wol/__init__.py

```diff
@@ -119,40 +123,133 @@ def setup_events(player: int, locked_locations: typing.List[str], location_cache
 
 
 def get_excluded_items(multiworld: MultiWorld, player: int) -> Set[str]:
-    excluded_items: Set[str] = set()
-
+    excluded_items: Set[str] = set(get_option_value(multiworld, player, 'excluded_items'))
     for item in multiworld.precollected_items[player]:
         excluded_items.add(item.name)
-
-    excluded_items_option = getattr(multiworld, 'excluded_items', [])
-
-    excluded_items.update(excluded_items_option[player].value)
+    locked_items: Set[str] = set(get_option_value(multiworld, player, 'locked_items'))
+    # Starter items are also excluded items
+    starter_items: Set[str] = set(get_option_value(multiworld, player, 'start_inventory'))
+    item_table = get_full_item_list()
+    mutation_count = get_option_value(multiworld, player, "include_mutations")
+    strain_count = get_option_value(multiworld, player, "include_strains")
+
+    # Exclude Primal Form item if option is not set
+    if get_option_value(multiworld, player, "kerrigan_primal_status") != 5:
+        excluded_items.add("Primal Form (Kerrigan)")
```

Comment:
This item should be the energy regen upgrade (and if the given option is set, then it connects skins). That'd make logic for Supreme easier to balance (requiring this or high enough level for standard tactics)

Reply:
I strongly dislike this. I agree Supreme logic would be easier, but I think the skin change + energy regen are an iconic combo of HotS in vanilla, I would not like to take them apart.

Reply:
If you'd have always human Kerrigan and Supreme on, you'd have problems with Standard tactics - The game should fail to generate then (ability spam should be the standard tactics). Engine-side it's a separate upgrade.

I'd separate actual upgrades (that add abilities or change stats) from skins